resources/lib/tokenresolver.py

```diff
@@ -144,7 +144,7 @@ def login(self, refresh=False, token_variant=None):
         while login_json.get('errorCode') != 0:
             # Show localized login error messages in Kodi GUI
             message = login_json.get('errorDetails')
-            self._kodi.log_notice('Login failed: %s' % message)
+            self._kodi.log('Login failed: {message}', message=message)
```

Comment:
```
07:53:56.787 T:139893811599104   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <type 'exceptions.TypeError'>
                                            Error Contents: log() got multiple values for keyword argument 'message'
                                            Traceback (most recent call last):
                                              File "/storage/.kodi/addons/plugin.video.vrt.nu/resources/lib/addon_plugin.py", line 8, in <module>
                                                run(sys.argv)
                                              File "/storage/.kodi/addons/plugin.video.vrt.nu/resources/lib/addon.py", line 248, in run
                                                plugin.run(argv)
                                              File "/storage/.kodi/addons/script.module.routing/lib/routing.py", line 130, in run
                                                self._dispatch(self.path)
                                              File "/storage/.kodi/addons/script.module.routing/lib/routing.py", line 141, in _dispatch
                                                view_func(**kwargs)
                                              File "/storage/.kodi/addons/plugin.video.vrt.nu/resources/lib/addon.py", line 220, in play_id
                                                VRTPlayer(kodi).play(dict(video_id=video_id, publication_id=publication_id))
                                              File "/storage/.kodi/addons/plugin.video.vrt.nu/resources/lib/vrtplayer.py", line 293, in play
                                                stream = _streamservice.get_stream(video)
                                              File "/storage/.kodi/addons/plugin.video.vrt.nu/resources/lib/streamservice.py", line 194, in get_stream
                                                stream_json = self._get_stream_json(api_data, roaming)
                                              File "/storage/.kodi/addons/plugin.video.vrt.nu/resources/lib/streamservice.py", line 151, in _get_stream_json
                                                playertoken = self._tokenresolver.get_playertoken(token_url, token_variant='ondemand', roaming=roaming)
                                              File "/storage/.kodi/addons/plugin.video.vrt.nu/resources/lib/tokenresolver.py", line 58, in get_playertoken
                                                xvrttoken = self.get_xvrttoken(token_variant=xvrttoken_variant)
                                              File "/storage/.kodi/addons/plugin.video.vrt.nu/resources/lib/tokenresolver.py", line 81, in get_xvrttoken
                                                token = self.login(token_variant=token_variant)
                                              File "/storage/.kodi/addons/plugin.video.vrt.nu/resources/lib/tokenresolver.py", line 147, in login
                                                self._kodi.log('Login failed: {message}', message=message)
                                            TypeError: log() got multiple values for keyword argument 'message'
                                            -->End of Python script error report<--
07:53:56.797 T:139894709573312   DEBUG: ------ Window Init (DialogNotification.xml) ------
```